lib/test.js

```diff
@@ -21,7 +21,7 @@ var Test = function(options) {
   }
   
   this.web3 = new Web3();
-  this.web3.setProvider(this.sim.provider());
+  this.web3.setProvider(this.sim.provider(options));
```

Comment:
Alright makes sense, it will be available as providerOptions.

Thanks for the PR btw, It seems you were using the develop branch which can often be broken. I merged a bunch of new stuff this morning in preparation for 2.3.0. Includes more fixes to the testing functionality.